lib/modules/dosomething/dosomething_campaign/dosomething_campaign_hot_shares/dosomething_campaign_hot_shares.cron.inc

```diff
@@ -0,0 +1,36 @@
+<?php
+
+/**
+ * @file
+ * Provides cron job command for dosomething_campaign_problem_shares.module.
+ */
+
+/**
+ * Implements hook_cron().
+ */
+function dosomething_campaign_hot_shares_cron() {
+  $results = db_query("SELECT nid FROM node WHERE type ='campaign'");
+
+  // Get the relevant paths
+  $path = dosomething_helpers_application_path('/lib/modules/dosomething/dosomething_campaign/dosomething_campaign_hot_shares');
+  $font_path = dosomething_helpers_application_path('/lib/modules/dosomething/dosomething_campaign/fonts');
+  $image_location = dosomething_helpers_application_path('/html/sites/default/files');
+
+  // Each active campaign, call shell script to create problem share images.
+  foreach($results as $result) {
+    $campaign = Campaign::get($result->nid)[0];
+    $node = node_load($result->nid);
```

Comment:
Did a little benchmarking to see. When running on my local development machine (so not necessarily a scientific test), the total time spent on making hot share images was `22.258s` and the total time making the problem shares was `25.073s`, so they're in the same ballpark.

From (again, unscientific) benchmarking, it looks like loading an individual campaign with `Campaign::get` takes about `0.005s` and loading a campaign node with `node_load` takes about `0.000036s`. From what I can tell, the overwhelming bulk of the time is spent running the Imagemagick shell script.